### ISSUE TITLE
docker `latest` tag has wrong version format $-react-$-js-$

### ISSUE THREAD

**Issue by lanerussell** [vector-im/element-web#17325]
### Description

Element version in element-web does not appear as normalized, i.e. 1.7.28

### Steps to reproduce

- View Help & About in element-web settings

Describe how what happens differs from what you expected.

Element version in element-web should appear as normalized, i.e. 1.7.28

Logs being sent: no

![1](https://user-images.githubusercontent.com/20979879/118494865-31da7380-b6e8-11eb-979d-b78c28a83a77.png)

### Version information

- **Platform**: web (in-browser)

For the web app:

- **Browser**: Firefox 88.0.1
- **OS**: Ubuntu
- **URL**: (private) Element version: `93e6abefedef-react-93e6abefedef-js-93e6abefedef`

**Comment by t3chguy** [comment]
You are on a develop version, so the version is made up of the git hashes at the time of the build.
Release versions are exactly as you desire, 1.7.27: https://app.element.io/version

**Comment by lanerussell** [comment]
Apologies, I incorrectly assumed the Docker tag "latest" would equate to the latest stable release. I've set my docker-compose file to vectorim/element-web:v1.7.28, and the version appears as expected. Perhaps what I should do is put in a request for a "stable" docker tag. Thanks for your help!

**Comment by t3chguy** [comment]
Hmm, I don't believe that is intentional. There is the `develop` tag for that
The `latest` build is indeed a build of `master` - 
![image](https://user-images.githubusercontent.com/2403652/118501071-463a6280-b720-11eb-98ae-eb54fd95359e.png)

It looks like this is flawed for the `master` branch
https://github.com/vector-im/element-web/blob/develop/scripts/docker-write-version.sh#L9-L20

**Comment by akoyaxd** [comment]
#12777
Seems like there already was an attempt to fix this. But can someone confirm, that it was once working? For last few month it was definitely broken.

**Comment by t3chguy** [comment]
```
root@nexus ~# docker run -it --rm vectorim/element-web:latest /bin/ash
Unable to find image 'vectorim/element-web:latest' locally
latest: Pulling from vectorim/element-web
540db60ca938: Already exists 
0ae30075c5da: Pull complete 
9da81141e74e: Pull complete 
b2e41dd2ded0: Pull complete 
7f40e809fb2d: Pull complete 
758848c48411: Pull complete 
c7a2f01303f1: Pull complete 
c2146db7bf71: Pull complete 
e386411f0976: Pull complete 
Digest: sha256:7f8afd72ba6b98b586a94a23e576be6ada0554cc4d323effa6e67569d1320291
Status: Downloaded newer image for vectorim/element-web:latest
/ # ls /app
bundles                            i18n                               mobile_guide                       themes
config.json                        img                                olm.91fced0.wasm                   usercontent
contribute.json                    index.html                         olm_legacy.js                      vector-icons
decoder-ring                       jitsi.html                         opus-decoderWorker.min.6fe2ced.js  version
decoderWorker.min.1325b88.wasm     jitsi_external_api.min.js          opus-encoderWorker.min.b4c5d1b.js  wave-encoderWorker.min.945cec9.js
decoderWorker.min.wasm             manifest.json                      static                             welcome
fonts                              media                              sw.js                              welcome.html
/ # cat /app/version
6d46ed808231-react-6d46ed808231-js-6d46ed808231
```

**Comment by lanerussell** [comment]
This appears to have been resolved as of 1.8.0.

**Comment by turt2live** [comment]
This is because we've been doing manual builds. The automated builds still haven't been fixed.
